src/k8s/requests/v2v/startV2VvmwareController.js

```diff
@@ -53,8 +85,12 @@ export const startV2VVMWareController = async ({ namespace }, { k8sGet, k8sCreat
           ...nextResult,
         };
       },
-      Promise.resolve({ activeDeployment, namespace, name })
+      Promise.resolve({ activeDeployment, kubevirtVmwareConfigMap, namespace, name })
     );
+  } finally {
+    // eslint-disable-next-line no-console
+    console.log(`startV2VVMWareController for "${namespace}" namespace finished.`);
```

Comment:
IMO we should show this message only if the particular call deployed the controller and was not skipped because of the same version

Reply:
Ok, this can add extra information in connection to the "initial" message